.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+language: python
+python:
+  - "3.4"
```

Comment:
Don't we want to test it using more Python versions? (preferably including newer ones?)